easybuild/easyconfigs/a/ArmComputeLibrary/ArmComputeLibrary-23.08-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'SCons'
+
+name = 'ArmComputeLibrary'
+version = '23.08'
+
+homepage = 'https://github.com/ARM-software/ComputeLibrary'
+description = """The Arm Compute Library is a collection of low-level machine learning functions optimized for
+ Arm® Cortex®-A, Arm® Neoverse® and Arm® Mali™ GPUs architectures."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = ['https://github.com/ARM-software/ComputeLibrary/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['62f514a555409d4401e5250b290cdf8cf1676e4eb775e5bd61ea6a740a8ce24f']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('SCons', '4.6.0'),
+]
+
+prefix_arg = 'install_dir='
+
+buildopts = "os=linux arch=armv8a build=native multi_isa=1 "
+buildopts += "Werror=0 debug=0 neon=1 opencl=0 embed_kernels=0 "
+buildopts += "fixed_format_kernels=1 openmp=1 cppthreads=0 "
```

Comment:
from https://github.com/pytorch/pytorch/blob/main/.ci/docker/common/install_acl.sh#L13-L16

in particular, `arch=armv8a multi_isa=1` should be more generic without loosing functionality/performance: 

https://github.com/ARM-software/ComputeLibrary/blob/de7288cb71e6b9190f52e50a44ed68c309e4a041/docs/user_guide/library.dox#L567-L578

benchmarks on a64fx compared to `arch=armv8.2-a-sve` didn't show any difference